src/core/storage.py

```diff
@@ -0,0 +1,60 @@
+import pathlib
+import json
+from core.vehicle_list import VehicleList
+from collections.abc import Mapping, Sequence
+
+
+class JSONSerializer(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, Mapping):
+            serialized = {}
+            for k, v in o.items():
+                serialized[k] = self.default(v)
+        elif isinstance(o, Sequence):
+            serialized = []
+            for v in o:
+                serialized.append(self.default(v))
+        else:
+            if hasattr(o, "Serializable"):
+                serialized = o.Serializable()
+            else:
+                serialized = o
+
+        return serialized
+
+
+class DiskStorage(object):
+    def __init__(self, id):
+        self.customer_id = id
+        root_dir = pathlib.Path(__file__).parent.parent.parent.absolute()
+        self.storage_dir = root_dir / "storage"
+        self.storage_dir.mkdir(parents=True, exist_ok=True)
+
+    def getCustomerDir(self):
+        directory = self.storage_dir / f"{self.customer_id}"
+        directory.mkdir(parents=True, exist_ok=True)
+        return directory
+
+    def getVehicleFile(self, id):
+        return self.getCustomerDir() / f"{id}.json"
```

Comment:
Would recommend something like `werkzeug.secure_filename()` for any path generated from user input.